colossus/src/main/scala/colossus/service/ServiceServer.scala

```diff
@@ -87,7 +87,8 @@ object ServiceConfig {
 }
 
 trait RequestFormatter[I] {
-  def format(request: I): String
+  def formatRequest(request: I): String
+  def formatError(error: Throwable): String
 }
```

Comment:
An alternative to this would be an `ErrorFormatter` trait that contains its own method for formatting `Throwable`s separate from the request formatter.